app/src/main/res/layout/activity_item_detail.xml

```diff
@@ -24,6 +24,14 @@
             app:layout_scrollFlags="scroll|exitUntilCollapsed"
             app:toolbarId="@+id/toolbar">
 
+            <ImageView
+                android:id="@+id/bgHeader"
```

Comment:
you can make the name a little bit friendly. bgHeader stands for backgroundHeader?

Reply:
yeap!! I was so busy and just put the firt thing in my head!! jeje sorry!